src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-play-video-overlay/main.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+/**
+ * @class MediaPlayVideoOverlay
+ * Class which shows overlays for playing videos
+ * @constructor
+ */
+define([
+    'services/sulumedia/media-manager'
+], function(MediaManager) {
+
+    'use strict';
+
+    var namespace = 'sulu.media-play-video.',
+
+        defaults = {
+            instanceName: ''
+        },
+
+        constants = {
+            loadingClass: 'loading',
+            loaderClass: 'media-play-video-loader',
+            playVideoId: 'media-play-video-id'
+        },
+
+        templates = {
```

Comment:
also here we have an extension which handles template, defaults and translations. http://docs.sulu.io/en/latest/bundles/admin/javascript-hooks/defaults.html